hazelcast-jet-core/src/main/java/com/hazelcast/jet/stream/impl/StreamUtil.java

```diff
@@ -49,6 +50,10 @@ public static String uniqueMapName() {
         return MAP_PREFIX + randomName();
     }
 
+    public static String uniqueCacheName() {
```

Comment:
we can just get rid of these unique* methods and prefixes, as they're no longer used.